src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -529,48 +556,85 @@ protected override void StopProcessing()
         /// </summary>
         private MemoryStream _ms;
 
+        private StringWriter _sw;
+
+        private Serializer _cli_serializer;
+
         private void CreateMemoryStream()
         {
-            // Memory Stream
-            _ms = new MemoryStream();
-
-            // We use XmlTextWriter originally:
-            //     _xw = new XmlTextWriter(_ms, null);
-            //     _xw.Formatting = Formatting.Indented;
-            // This implies the following settings:
-            //  - Encoding is null -> use the default encoding 'UTF-8' when creating the writer from the stream;
-            //  - XmlTextWriter closes the underlying stream / writer when 'Close/Dispose' is called on it;
-            //  - Use the default indentation setting -- two space characters.
-            //
-            // We configure the same settings in XmlWriterSettings when refactoring this code to use XmlWriter:
-            //  - The default encoding used by XmlWriterSettings is 'UTF-8', but we call it out explicitly anyway;
-            //  - Set CloseOutput to true;
-            //  - Set Indent to true, and by default, IndentChars is two space characters.
-            //
-            // We use XmlWriterSettings.OmitXmlDeclaration instead of XmlWriter.WriteStartDocument because the
-            // xml writer created by calling XmlWriter.Create(Stream, XmlWriterSettings) will write out the xml
-            // declaration even without calling WriteStartDocument().
-
-            var xmlSettings = new XmlWriterSettings();
-            xmlSettings.Encoding = Encoding.UTF8;
-            xmlSettings.CloseOutput = true;
-            xmlSettings.Indent = true;
-
-            if (As.Equals("Stream", StringComparison.OrdinalIgnoreCase))
+            if (string.IsNullOrEmpty(Format))
```

Comment:
Given that I recommended using an enum value, this should also be a switch.

However, I'd like to point out so you're aware for the future -- this probably isn't the best practice when working with parameters like this. What you're doing here is basically "was this parameter provided at all", which is better done with:
```csharp
if (MyInvocation.BoundParameters.ContainsKey(nameof(Format)))
```